app/src/main/proto/encrypted_tokens.proto

```
@@ -0,0 +1,9 @@
+syntax = "proto3";
```

Buf (1.55.1)

Files must have a package defined.

(PACKAGE_DEFINED)